docs/running-on-yarn.md

```diff
@@ -528,6 +519,25 @@ To use a custom metrics.properties for the application master and executors, upd
   </td>
   <td>1.6.0</td>
 </tr>
+<tr>
+  <td><code>spark.yarn.executor.bindAddress</code></td>
+  <td><code>(executor hostname)</code></td>
+  <td>
+  Hostname or IP address where to bind listening sockets in YARN cluster mode.
```

Comment:
Why `YARN cluster mode`? The change affects client mode as well, doesn't it?

Reply:
Hmm true, will benefit `client` mode as well